examples/react-hooks/default-theme/src/App.tsx

```diff
@@ -108,6 +109,9 @@ export function App() {
           />
 
           <SearchBox placeholder="Search" autoFocus />
+          <div className="Stats">
+            <Stats />
```

Comment:
while this is great to test it out, we probably should revert it until released, as these examples are linked from eg. codesandbox. If we add something new that's not yet released, it will break.

We probably should change the type of links we use (latest released tag maybe for example), but for now master causes this issue